src/django/requirements.txt

```diff
@@ -27,4 +27,5 @@ thefuzz==0.19.0
 Unidecode==1.0.23
 xlrd==1.2.0
```

Comment:
The OAR team confirmed that we can drop support for .xls files so we can remove `xlrd` from our requirements.